Files/UserControls/Widgets/Bundles.xaml

```diff
@@ -318,6 +324,15 @@
                                                                 <FontIcon FontFamily="{StaticResource CustomGlyph}" Glyph="&#xF106;" />
                                                             </MenuFlyoutItem.Icon>
                                                         </MenuFlyoutItem>
+                                                        <MenuFlyoutItem
+                                                            x:Uid="SideBarOpenInNewPane"
```

Comment:
```suggestion
                                                            x:Uid="BundlesOpenInNewPane"
```
Can you add a new string `BundlesOpenInNewPane`?

Reply:
@yaichenbaum all done 👍